whylog/tests/utils.py

```diff
@@ -0,0 +1,26 @@
+import os.path
+
+path_test_files = ['whylog', 'tests', 'tests_config', 'test_files']
+
+
+def get_config_path():
+    path = os.path.join(*path_test_files)
+    return os.path.join(path, 'config.yaml')
+
+
+def create_yaml_paths():
+    path = os.path.join(*path_test_files)
+    config_path = os.path.join(path, 'config.yaml')
+    parsers_path = os.path.join(path, 'parsers.yaml')
+    rules_path = os.path.join(path, 'rules.yaml')
+    log_type_path = os.path.join(path, 'log_type.yaml')
+    return config_path, parsers_path, rules_path, log_type_path
+
+
+def create_config():
```

Comment:
Create config function creates mock content of config file and path to this file. In my opinion there are nothing dirty with creating two objects. Every time when SB will need config file, he will need path to the file too. This is only simple function for tests. It isn't so much important.